### ISSUE TITLE
Dataset support : ImageNet

### ISSUE THREAD

**Issue by rapidrabbit76** [rapidrabbit76/Classification-For-Everyone#12]
# ImageNet 데이터 지원 

# TODO 

- [x] imageNet 학습용 pytorch dataset 
- [x] imageNet pl.datamodule

**Comment by zkdlfrlwl2** [comment]
이건 지금 dataframe 까지만 만들어 놓은 상태지 ?

**Comment by rapidrabbit76** [comment]
> 이건 지금 dataframe 까지만 만들어 놓은 상태지 ?

ㅇㅇ 

train, validation, test(2012데이터셋의 val) 이렇게 3개로 parqeut으로 만들어 놨지 

저번에 줬던 스니핏 기반으로 컬럼은 

index, image, label 

3개로 구성

**Comment by rapidrabbit76** [comment]
> 이건 지금 dataframe 까지만 만들어 놓은 상태지 ?

https://d2.naver.com/helloworld/3773258 참고해서 
pytorch parqeut dataset은 만들어 놨음 
테스트도 해봤고 성능도 비슷하게 나오는거 확인 했는데 뭔가 마음에 안들어서 ... 
아마 공유 메모리 작업이랑 레이스 컨디션 없애는 작업 안하고 우회하는 방향으로 해서 내맘에 안드는듯...
 
Ray로 Dataloader를 다 뜯어고쳐 볼까도 고민 중 
이건 시간이 없어서 당분간은 힘들거 같고 
dataset 만들어 둔거는 조금 더 다음어서 따로 repo 만들고 pypi 에 올려보려고 
만들어서 pip로 받아서 하잨ㅋㅋㅋ

**Comment by rapidrabbit76** [comment]
#21 으로 ImageNet dataset 추가 함 

이슈는 닫겠습니다.
